### PR TITLE
updates for text

### PR DESCRIPTION
**Pull request by jordansread**
@lindsaycarr can you review?

### CODE REVIEW

File: data/siteText.yaml [hunk 1]
```diff
@@ -14,20 +14,22 @@ welcome-p3: >-
...
-walleyeBass-intro: >- 
-  The lake conditions that are best for one species are generally not the best for the other. 
-  Temperature is a key characteristic in determining the suitability of a lake for walleye or largemouth bass.
+walleyeBass-l1: Water temperature is key characteristic determining the suitability of a lake for different species of fish, such as walleye or largemouth bass.
 walleyeBass-interactive-setup: Can you draw the relationship between these two fish and lake temperature?
-walleyeBass-l1: Walleye prefer cool water
-walleyeBass-l2: LMB prefer warm water
+walleyeBass-l2: In general, walleye naturally reproduce more successfully in cool lakes, and largemouth bass populations are most abundant in warm lakes.
+walleyeBass-l3: >- 
+  Cool lakes that are best for walleye are the worst places for largemouth bass.
```
Comment (lindsayplatt) [inline data/siteText.yaml:28]: "the worst" sounds a bit extreme, even if it's true

Comment (lindsayplatt) [inline data/siteText.yaml:28]: maybe `are poor locations for largemouth bass`?

File: data/siteText.yaml [hunk 2]
```diff
@@ -14,20 +14,22 @@ welcome-p3: >-
...
-  The lake conditions that are best for one species are generally not the best for the other. 
-  Temperature is a key characteristic in determining the suitability of a lake for walleye or largemouth bass.
+walleyeBass-l1: Water temperature is key characteristic determining the suitability of a lake for different species of fish, such as walleye or largemouth bass.
 walleyeBass-interactive-setup: Can you draw the relationship between these two fish and lake temperature?
-walleyeBass-l1: Walleye prefer cool water
-walleyeBass-l2: LMB prefer warm water
+walleyeBass-l2: In general, walleye naturally reproduce more successfully in cool lakes, and largemouth bass populations are most abundant in warm lakes.
+walleyeBass-l3: >- 
+  Cool lakes that are best for walleye are the worst places for largemouth bass. 
+  At the same time, the warm lakes that are best for finding high numbers of largemouth bass have the worst conditions for walleye.
```
Comment (lindsayplatt) [inline data/siteText.yaml:29]: same feeling towards the use of "the worst" here

Comment (jordansread) [inline data/siteText.yaml:29]: I'm deferring to the fisheries biologist for this text, so we can iterate with her on the specific words used.

Comment (lindsayplatt) [inline data/siteText.yaml:29]: Ok, at a later date or before this is merged?

Comment (jordansread) [inline data/siteText.yaml:29]: later date

**Pull request merged**